src/test/java/hw3/steps/ServicePageSteps.java

```diff
@@ -0,0 +1,47 @@
+package hw3.steps;
+
+import hw3.pages.ServicePage;
+import org.openqa.selenium.WebElement;
+import org.testng.Assert;
+
+public abstract class ServicePageSteps {
```

Comment:
all assertions must get as parameter WebElement/List<WebElement> not PO. Refactor all assertions.

Moreover. For what this class? Use BaseTest to common assertions or create special methods in PO.